spec/deployer_spec.rb

```diff
@@ -102,6 +102,31 @@
       end
     end
 
+    context 'with master as commit' do
+      let(:commit) { 'master' }
+      let(:deployer) do
+        Deployer.new(
+          Config.repositories.find do |repo|
+            repo.name == 'sidekiq-monitoring'
+          end,
+          commit: commit
+        )
+      end
+
+      it 'deploys  the current master reference instead of master' do
```

Comment:
local master vs. master on origin?

Reply:
when you do a `depctl deploy gapfish --commit master --tag latest`, the image with tag `latest` is going to be deployed. that's a problem though when you want to track down the exact version.

this pull request changes deployer to be able to do `depctl deploy gapfish --commit master` and then deployer deploys the current image with the current master commit hash. so this results for instance to be the image `master-80f5aedde86d7e83f7dbcec3003f9dff84cfa67f`.